### PR TITLE
FIRECERT-2157-SecureStorage Securestorage manage test cases

### PR DESCRIPTION
**Pull request by anjalimukundan**

### CODE REVIEW

File: cypress/fixtures/modules/securestorage.json [hunk 1]
```diff
@@ -272,19 +272,130 @@
...
     "VALUE_TTL": "authTestTokenValueTTL",
     "SETFORAPP_DEVICE": {
-        "appId": "refui",
+        "appId": "boltextn",
         "scope": "device",
         "key": "authTestTokenDevice",
         "value": "VGhpcyBub3QgYSByZWFsIHRva2VuLgo="
     },
+    "SETFORAPP_INVALID_SCOPE": {
+        "appId": "boltextn",
```
Comment (neeradanelxsi) [inline cypress/fixtures/modules/securestorage.json:281]: boltextn is ripple specific. this cannot go in FCS

File: cypress/TestCases/FireboltCertification/Manage/SecureStorageManage.feature [hunk 2]
```diff
@@ -6,52 +6,140 @@ Feature: SecureStorage_Manage
...
+        And 'Firebolt' platform responds to '1st party app' for 'null for clearing stored value for an app'
+        And 1st party app invokes the 'Firebolt' API to '<API_Key>'
+        And 'Firebolt' platform responds to '1st party app' for 'null for getting stored value'
         When 1st party app invokes the 'Firebolt' API to '<Set_API_Key>'
-        Then 'Firebolt' platform responds to '1st party app' for '<Set_API_Key>'
+        Then 'Firebolt' platform responds to '1st party app' for 'invalid params for setting a data value in securestorage'
         When 1st party app invokes the 'Firebolt' API to '<API_Key>'
         Then 'Firebolt' platform responds to '1st party app' with 'null for getting stored value'
 
         Examples:
```
Comment (neeradanelxsi) [inline cypress/TestCases/FireboltCertification/Manage/SecureStorageManage.feature:80]: for negative test cases, why do we need to have all the extra steps ? wouldn't this throw error irrespective of the extra steps ?

Comment (anjalimukundan) [reply cypress/TestCases/FireboltCertification/Manage/SecureStorageManage.feature:76]: Simplified the TCs with 2 steps.
Invoking the api with wrong param and validating with error.

File: cypress/fixtures/modules/securestorage.json (hunk 1 above)
Comment (anjalimukundan) [reply cypress/fixtures/modules/securestorage.json]: updated to "comcast.test.firecert" and moved to configModule

File: cypress/TestCases/FireboltCertification/Manage/SecureStorageManage.feature [hunk 3]
```diff
@@ -6,52 +6,117 @@ Feature: SecureStorage_Manage
...
+            | removeForApp with device scope  | remove secure data value for an app with scope device   | set secure data value for an app with scope device1  | set secure data value for an app with scope device2  | get stored value for an app for authTestTokenDevice1 with scope device   | get stored value for an app for authTestTokenDevice2 with scope device   | expected value for authTestTokenDevice2 stored data in securestorage  | expected value for authTestTokenDevice1 stored data in securestorage  | clear stored value with scope as device for an app  |
+            | removeForApp with account scope | remove secure data value for an app with scope account1 | set secure data value for an app with scope account1 | set secure data value for an app with scope account2 | get stored value for an app for authTestTokenAccount1 with scope account | get stored value for an app for authTestTokenAccount2 with scope account | expected value for authTestTokenAccount2 stored data in securestorage | expected value for authTestTokenAccount1 stored data in securestorage | clear stored value with scope as account for an app |
```
Comment (neeradanelxsi) [inline cypress/TestCases/FireboltCertification/Manage/SecureStorageManage.feature:46]: we cannot use device1/ device2/ account1/ account2 when we are not actually using device1 /account1 etc. All we are testing is on a single device with a single app .

Comment (anjalimukundan) [reply cypress/TestCases/FireboltCertification/Manage/SecureStorageManage.feature:50]: We have added TCs only for single device and account.
device1, device2 ... etc were used for authenticationValue1 and authenticationValue2.
Renamed the params and validation objects.

**Review by neeradanelxsi — APPROVED** [review]

**Review by jbigel — APPROVED** [review]

**Pull request merged**